### PR TITLE
Feature/123 make learning clearer

### LINKED ISSUE #123: Improve learning UI to make usage clearer

### PR DESCRIPTION
**Pull request by Nitwix**
Wraps images from learning UI inside a card view to have rounded corners and a border that indicates with which element to interact.

The idea was that first the image to sort has a green border to indicate that you should interact with it. Then, once you start dragging it, the target categories have the green border, to indicate you should drag the image on one of the targets. If it is wrong, the border of the category turns red to indicate a mistake.

This could not be implemented yet because of a lack of time on my part. I'll add an issue so that we can implement this in a future sprint.

### CODE REVIEW

File: app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt [hunk 1]
```diff
@@ -222,29 +227,29 @@ class LearningFragment: Fragment() {
         return true
     }
 
+    private fun onImageToSortTouched(view: View, event: MotionEvent): Boolean {
+        val clipDataLabel = "My Clip Data"
```
Comment (wattsecs) [inline app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt]: Might not be very useful, but for the sake of completness maybe make it related to the actual data?

**Review by wattsecs — APPROVED** [review]
One very minor comment, otherwise all good

**Comment by Nitwix** [comment]
It seems there is a compilation error. Fix in progress

File: app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt (hunk 1 above)
Comment (Nitwix) [reply app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt]: yep. this still dates from the prototyping phase of this fragment. Will fix

**Review by NicolasVial — APPROVED** [review]
Way clearer, good job.

**Pull request merged**